dragonflybsd/DragonFlyBSDMachine.c

```diff
@@ -105,7 +105,7 @@ Machine* Machine_new(UsersTable* usersTable, uid_t userId) {
    }
 
    len = sizeof(kernelFScale);
-   if (sysctlbyname("kern.fscale", &kernelFScale, &len, NULL, 0) == -1) {
+   if (sysctlbyname("kern.fscale", &kernelFScale, &len, NULL, 0) == -1 || kernelFScale <= 0) {
```

Comment:
Note: I have no tools to test if this builds in DragonFlyBSD, but I suspect there is a `this->` specifier missing. It should have been `this->kernelFScale` to prevent compile error.

Reply:
Good catch. Based on Looking at the Code™ it's undeniable, this could never compile. Please go ahead and fix this.